src/main/java/mate/jdbc/exeption/DataProcessingException.java

```diff
@@ -0,0 +1,7 @@
+package mate.jdbc.exeption;
```

Comment:
Here's a minor typo in you package name:
```suggestion
package mate.jdbc.exception;
```